Group dependencies for submodules (#67)
facebookresearch/AugLy: augly/text/README.md

# Text

## Installation   

> If you plan to use the text augmentations from AugLy, you can install using the following command:   

Reviewer: ```suggestion
If you would like to use the text augmentations, please install AugLy using the following command:
```
